src/test/java/io/vavr/collection/euler/Utils.java

```diff
@@ -115,4 +115,15 @@ static boolean isPalindrome(String val) {
     static boolean isPalindrome(int val) {
         return isPalindrome(Long.toString(val));
     }
+
+    static Stream<Long> pentagonal() {
```

Comment:
The t._1 is only based on t._2.
_2 is 1 2 3 4
So they can be split into Stream.from and map.